content/java/concepts/date/terms/equals/equals.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: '.equals()'
+Description: 'A method designed to check if the invoking date object
+contains a date and time that is equal to the one specified by date object'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Calendar'
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.equals()`** is method of `Date` class. It compares the current date object to the specified object based on millisecond difference. It will returns **`true`** if the invoking date object contains a date that is equal to the one specified by date object. Otherwise, it returns **`false`**.
+
+## Syntax
+
+```pseudo
+boolean equals(Object date)
+```
+>**Note:** The return type of the method is  `boolean` so this method will return either `true` or `false`.
+
+## Example
+
+In this example, the `JavaDateEqualsExample` class demonstrates the use of `.equals()`method. 
+
+```java
+import java.util.Date;
```

Comment:
update the example to start comments with a capital and 2 spaces for indentation (tabs are not used in docs)